src/parser/parsed_data/create_table_info.cpp

```diff
@@ -29,4 +29,26 @@ unique_ptr<CreateInfo> CreateTableInfo::Copy() const {
 	return std::move(result);
 }
 
+string CreateTableInfo::ToString() const {
+	string ret = "";
+
+	ret += "CREATE TABLE " + table;
+	if (query != nullptr) {
```

Comment:
There's already code for converting a `TableCatalogEntry` to SQL - see `TableCatalogEntry::ToSQL`. Could we unify this with that?